### ISSUE TITLE
Add a way to copy data easily

### ISSUE THREAD

**Issue by robinst** [rsocket/rsocket-chrome-devtools#35]
## Motivation

Sometimes I need to copy the data part of a frame from the inspector to paste elsewhere. Currently, the data is shown in 3 places, for example:

<img width="1053" alt="Screenshot 2024-01-25 at 12 40 14 pm" src="https://github.com/rsocket/rsocket-chrome-devtools/assets/16778/aa0b8678-ead1-4c42-be38-c3b7c9b3dd47">

But they all have problems:
* The top part escapes `"` characters, so I have to unescaped them after pasting
* The `data` section shows `Object`, so selecting that and pasting it loses data
* The bottom right section contains the data unescaped, but when copying and pasting that, it includes the line breaks :(

## Desired solution

It would be great if there was a good way to copy data. Some ideas to fix this:

* Add a "Copy" button to the middle data section
* Make the copied content of the bottom right section not include line breaks (not sure how easy that is)

**Comment by GiganticRoss** [comment]
@SerCeMan 🙏

**Comment by robinst** [comment]
> * Add a "Copy" button to the middle data section

I've looked into this but getting this error: `The Clipboard API has been blocked because of a permissions policy applied to the current document.`

I've tried adding a `clipboardWrite` permission to the manifest, but that didn't help. It might be possible by sending a message to a content script instead and then writing it to the clipboard from there, but that seems like a lot of work for this, so not gonna work on that.

> * Make the copied content of the bottom right section not include line breaks (not sure how easy that is)

Implemented this in
* https://github.com/rsocket/rsocket-chrome-devtools/pull/37

It's available in the most recent release 0.0.5.
